src/org/scada_lts/dao/migration/mysql/V2_7__AnonymousUser.java

```diff
@@ -0,0 +1,42 @@
+package org.scada_lts.dao.migration.mysql;
+
+import com.serotonin.mango.Common;
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+import org.flywaydb.core.api.migration.BaseJavaMigration;
+import org.flywaydb.core.api.migration.Context;
+import org.scada_lts.dao.DAO;
+import org.springframework.jdbc.core.ArgumentPreparedStatementSetter;
+import org.springframework.jdbc.core.JdbcTemplate;
+
+import java.sql.PreparedStatement;
+
+public class V2_7__AnonymousUser extends BaseJavaMigration {
```

Comment:
I would change the name to V2_7__ to avoid conflicts with other branches that have migration and will be merged in version 2.7